daemon/src.cmake

```diff
@@ -301,7 +301,7 @@ set(QCOMMONLIST
     ${ENGINE_DIR}/sys/con_common.cpp
 )
 
-if (NOT APPLE)
+if (NOT APPLE AND (USE_CURSES OR WIN32))
```

Comment:
What I wonder is if someone compiles curses from source or something to enable support.
Though I don't use either apple or win so I don't know if it's possible.